include/structs/dungeon_entity.h

```diff
@@ -418,6 +418,31 @@ enum VisualFlag
     VISUAL_FLAG_RUN_AWAY = 2
 };
 
+enum ExpMultiplier
+{
+    EXP_HALVED, // 0.5x when fainting a mon without using regular attacks
```

Comment:
I see the intention of this comment, though it may cause confusion because the standard A attack is usually called the "regular attack". Might be worth specifying something like "fainting a mon with only the regular attack and no moves".